### PR TITLE
Clarification in psa_attestation.md as part of OOB 5.12

### PR DESCRIPTION
**Pull request by GuyWi**
This PR addresses [IOTDOCS-972](https://jira.arm.com/browse/IOTDOCS-972).

### CODE REVIEW

**Review by AnotherButler — APPROVED** [review]
Thanks for the PR 👍

**Comment by AnotherButler** [comment]
ping @ashok-rao @avolinski

File: docs/api/security/psa_attestation.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## PSA initial attestation
 
-The PSA initial attestation service enables an application to prove a device's identity to a caller during the authentication process.
+The PSA initial attestation service enables an application to prove a device's identity to any server or device before communicating data, as part of the authentication process.
```
Comment (alekshex) [inline docs/api/security/psa_attestation.md]: my proposal:
The PSA initial attestation service enables an application to prove a device's identity to any server and application, as part of the authentication process.

**Comment by alekshex** [comment]
pong :)
wrote a comment. other than that seems ok to me

**Pull request merged**